app/views/application/_header.html.erb

```diff
@@ -1,10 +1,17 @@
 <div class="application-_header clearfix">
 
-<%# clearly this is just a temporary 💩 %>
-<div style="position: fixed; top: -1000px; left: -1000px;">
-<%= current_user.nickname %>
-<%= current_user.email %>
-<%= image_tag current_user.avatar_url %>
-</div>
+<header class="site-header clearfix" data-ux-header>
+  <div class="placeholder__logo"></div>
+  <div class="main-action-container clearfix">
+    <%= content_for :main_action %>
+  </div>
+  <div class="user-profile clearfix">
+    <div class="avatars-container clearfix">
+      <%= render partial: "users/avatar",
+                 object: current_user, as: :user,
```

Comment:
`users/_avatar.html.erb` is used as...

```
      <%= render partial: "users/avatar",
                 collection: board.collaborators, as: :user,
                 locals: { size: "tiny" } %>
```

... in https://github.com/catchup/catchup-web/pull/60/files#diff-641397f5b5168b3fed969d3a880d9358R13.

Do you think could it work anyway?